qutebrowser/config/configdata.py

```diff
@@ -657,6 +657,10 @@ def data(readonly=False):
              SettingValue(typ.Bool(), 'false'),
              "Whether to open windows instead of tabs."),
 
+            ('refresh-on-select', 
```

Comment:
This feature is commonly called lazy loading, so I'd say we should call the option `lazy-load` instead?